cf_xarray/accessor.py

```diff
@@ -461,6 +461,14 @@ def _getattr(
     try:
         attribute: Union[Mapping, Callable] = getattr(obj, attr)
     except AttributeError:
+        if getattr(
+            CFDatasetAccessor if isinstance(obj, DataArray) else CFDataArrayAccessor,
+            attr,
+            None,
+        ):
+            raise AttributeError(
+                f"{obj.__class__.__name__+'.cf'!r} object has no attribute {attr!r}"
+            )
```

Comment:
I've added this to try to clarify when `cf_xarray` only adds the attribute to a DataArray/Dataset or the attribute is not present in the underlying `xarray` object.

Reply:
nice.thanks!